src/V3Options.cpp

```diff
@@ -1472,6 +1472,7 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc, char
     DECL_OPTION("-y", CbVal, [this, &optdir](const char* valp) {
         addIncDirUser(parseFileArg(optdir, string(valp)));
     });
+    DECL_OPTION("-format-cxx-code", OnOff, &m_formatCxxCode);
```

Comment:
This option list should be in lexically sorted order, so please insert as such

Reply:
I went with approach to use already existing ``--no-decoration`` option, so no new option is needed. Removed.